test/built-ins/Array/prototype/filter/15.4.4.20-9-c-iii-28.js

```diff
@@ -6,11 +6,11 @@ es5id: 15.4.4.20-9-c-iii-28
 description: >
     Array.prototype.filter - return value of callbackfn is the global
     object
-includes: [fnGlobalObject.js]
 ---*/
 
+var global = this;
         function callbackfn(val, idx, obj) {
-            return fnGlobalObject();
+            return global;
```

Comment:
this whole test seems unnecessary. It's testing a `ToBoolean(global) === true`. That can be done with a simple `{}`.